paasta_tools/cli/cmds/validate.py

```diff
@@ -317,13 +328,27 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
-def validate_tron(service_path):
+def validate_tron(service_path, verbose=False):
```

Comment:
while we're here, we can be good scouts and leave this a better place than we found it and add some types:
```suggestion
def validate_tron(service_path: str, verbose: bool = False) -> bool:
```